buildSrc/src/main/java/Dependencies.kt

```diff
@@ -35,7 +35,7 @@ object Versions {
     // Note that android-components also depends on application-services,
     // and in fact is our main source of appservices-related functionality.
     // The version number below tracks the application-services version
-    // that we depend on directly for the fenix-megazord (and for it's
+    // that we depend on directly for the fenix-megazord (and for its
```

Comment:
Sorry, I could not resist removing this inappropriate apostrophe while I was in here...